edbdeploy/data/terraform/aws/environments/ec2/ec2.tf

```diff
@@ -448,7 +530,9 @@ resource "null_resource" "pg-set-hostname" {
         "/tmp/add_ips_to_hosts.sh ${var.cluster_name} pem ${join(" ", aws_instance.pem_server.*.private_ip)} >> /tmp/pem_hostnames.log 2>&1",
         "/tmp/add_ips_to_hosts.sh ${var.cluster_name} postgres ${join(" ", aws_instance.postgres_server.*.private_ip)} >> /tmp/postgres_hostnames.log 2>&1",        
         "/tmp/add_ips_to_hosts.sh ${var.cluster_name} barman ${join(" ", aws_instance.barman_server.*.private_ip)} >> /tmp/barman_hostnames.log 2>&1",
-        "/tmp/add_ips_to_hosts.sh ${var.cluster_name} pooler ${join(" ", aws_instance.pooler_server.*.private_ip)} >> /tmp/pooler_hostnames.log 2>&1"        
+        "/tmp/add_ips_to_hosts.sh ${var.cluster_name} pooler ${join(" ", aws_instance.pooler_server.*.private_ip)} >> /tmp/pooler_hostnames.log 2>&1",
+        "chmod a+x /tmp/add_nickname_to_shell.sh",        
+        "/tmp/add_nickname_to_shell.sh ${var.cluster_name} postgres ${join(" ", aws_instance.postgres_server.*.private_ip)} >> /tmp/pem_nicknames.log 2>&1"
```

Comment:
Log file name should be `postgres_nicknames.log`

Reply:
Thank you for reviewing.

FIxed in PER-42v3, ready for review and merge.